content/strategy-goals/strategy/cloud/security/index.md

```diff
@@ -22,7 +24,9 @@ These are goals that we have not started working on, but expect to begin work on
 
 ### 1. Secure Sourcegraph Cloud for SMBs
 
-**Problem:** The [Cloud SaaS team](../../../../departments/product-engineering/engineering/cloud/saas/index.md) team is working toward [enabling Sourcegraph Cloud as an alternative to on-premises deployments](../../../../departments/product-engineering/engineering/cloud/saas/index.md). To be confident that Sourcegraph Cloud is a secure alternative to Sourcegraph on-premises, we need a high degree of confidence in our security posture. We'll do this by improving visibility across our entire threat surface, as well as continuously and proactively ensuring the efficacy of our security controls.
+<!-- TODO: Can we replace the references to Cloud Saas  with Cloud Org?-->
```

Comment:
Can we replace the references to Cloud Saas with Cloud Org?

Reply:
Yes, replacing with Cloud org makes sense here.